docs/DeveloperGuide.adoc

```diff
@@ -372,7 +372,7 @@ _{More to be added}_
 Use case ends.
 
 [discrete]
-=== Use case 02: Add bookmark
+=== Use case: UC02 - Add bookmark
```

Comment:
Should we change this? (same for other use similar cases)
```suggestion
=== Use case: UC02 - Add bookmarks
```

Reply:
Hm it's ok to me! Since we are only adding one bookmark in this use case.